presto-spi/src/main/java/com/facebook/presto/spi/type/CharType.java

```diff
@@ -30,6 +30,7 @@
         extends AbstractVariableWidthType
 {
     public static final int MAX_LENGTH = 65_536;
+    public static final CharType CHAR = new CharType(MAX_LENGTH);
```

Comment:
Oh, sure. I am OK to add it just in the testcase. Seems all types have an constant in its classfiles. Just try to make CharType conform similar to other types. Maybe we could leave this as it is, and file another task, to remove them all?